lib/index.js

```diff
@@ -102,6 +125,24 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
 
   app.options(accountPath, cors());
 
+  app.get(
+    `${accountPath}/roles`,
+    ensureAuthenticated,
+    cors(),
+    asyncHandler(async (req, res) => {
+      /**
+       * @func getCapabilitiesForAnAccount
+       * @param {Request} req
+       * @param {Response} res
+       * @description uses the account param to get
+       * all the capabilities for an account
+       * @route /:account/roles
+       */
+      const {account: id} = req.params;
+      const record = await brAccount.getCapabilities({id});
+      res.status(200).json(record);
```

Comment:
No need for status 200 here.  I guess this was fixed elsewhere.

Reply:
fixed in the other PR I believe.